src/app/home-page/widget/starred-box/starred-box.component.scss

```diff
@@ -0,0 +1,28 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.list-events {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2; /* number of lines to show */
```

Comment:
There's a global style that truncates lines: https://github.com/dockstore/dockstore-ui2/blob/develop/src/styles.scss#L1189. In this case, you would do something like `truncate-text-2` to only show 2 lines